server/build/webpack.js

```diff
@@ -182,4 +183,10 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       return interpolateNames.get(this.resourcePath) || url
     }
   })
+  const config = getConfig(dir)
+  if (config.webpack) {
+    console.log('> Using Webpack config function defined in next.config.js.')
+    return config.webpack(webpackConfig, hotReload)
```

Comment:
Right, I changed the second param of `webpackConfig` to `{ hotReload, dev }` so we are more consistent with `createCompiler`.